src/app/components/applications/SSV/RegisterValidatorHome/components/DkgOperator/DkgOperator.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Grid from '@mui/material/Grid';
+import { IOperator } from '~app/common/stores/applications/SsvWeb/Operator.store';
+import {
+    useStyles,
+} from '~app/components/applications/SSV/RegisterValidatorHome/components/DkgOperator/DkgOperator.styles';
+
+const DkgOperator = ({ operator }: { operator: IOperator }) => {
+    const classes = useStyles({ operatorLogo: operator.logo, dkgEnabled: operator.dkg_address });
```

Comment:
Can dkgEnabled flag be true by the fact of real validation if it's domain name /ip address with optional port?
If value is not validated - then it's not dkg enabled

Reply:
done